README.md

```diff
@@ -2,17 +2,25 @@
 
 ## About
 
-This [Github repository](https://github.com/stencila/policies) is where we keep Stencila's policies, code of conduct, guidelines etc. Everything is available under a Creative Commons Attribution-ShareAlike [license](https://github.com/stencila/policies/blob/master/LICENSE.md). Just like our code, we think these policies and guidelines should be open source too!
+This [Github repository](https://github.com/stencila/policies) is where we keep Stencila's policies, code of conduct,
+guidelines etc. Everything is available under a Creative Commons Attribution-ShareAlike
+[license](https://github.com/stencila/policies/blob/master/LICENSE.md). Just like our code, we think these policies and
+guidelines should be open source too!
```

Comment:
This repo uses `@stencila/dev-config` and has a script `format": "prettier *.md --write"` which I ran before committing. This seems to use a different `md` formatting convention?

Reply:
ah, sorry I didn't notice.
The formatting is the same, but the line length has been constrained to 120 by my editor.
Prettier doesn't wrap prose [unless you enable it](https://prettier.io/docs/en/options.html#prose-wrap), as it can cause issues in certain cases.